Tests/xSharePoint/xSharePoint.xSPUserProfileSection.Tests.ps1

```diff
@@ -0,0 +1,299 @@
+
+[CmdletBinding()]
+param(
+    [string] $SharePointCmdletModule = (Join-Path $PSScriptRoot "..\Stubs\SharePoint\15.0.4693.1000\Microsoft.SharePoint.PowerShell.psm1" -Resolve)
+)
+Add-PSSnapin Microsoft.SharePoint.PowerShell -ea 0 
+
+$ErrorActionPreference = 'stop'
+Set-StrictMode -Version latest
+
+$RepoRoot = (Resolve-Path $PSScriptRoot\..\..).Path
+$Global:CurrentSharePointStubModule = $SharePointCmdletModule 
+    
+$ModuleName = "MSFT_xSPUserProfileSection"
+Import-Module (Join-Path $RepoRoot "Modules\xSharePoint\DSCResources\$ModuleName\$ModuleName.psm1")
+
+
+Describe "xSPUserProfileProperty" {
+    InModuleScope $ModuleName {
+        $testParams= @{
+           Name = "PersonalInformation"
+           UserProfileService = "User Profile Service Application"
+           DisplayName = "Personal Information"
+           DisplayOrder = 5000 
+        }
+        Import-Module $Global:CurrentSharePointStubModule -WarningAction SilentlyContinue 
+      #  $farmAccount = New-Object System.Management.Automation.PSCredential ("domain\username", (ConvertTo-SecureString "password" -AsPlainText -Force))
+        <#$testParamsUpdate = @{
+           Name = "PersonalInformation"
+           UserProfileService = "User Profile Service Application"
+           DisplayName = "Personal Information"
+           DisplayOrder = 5000        
+         }#>
+        
+        try { [Microsoft.Office.Server.UserProfiles] }
+        catch {
+            Add-Type @"
+                namespace Microsoft.Office.Server.UserProfiles {
+                public enum ConnectionType { ActiveDirectory, BusinessDataCatalog };
+                public enum ProfileType { User};
+                }        
+"@ -ErrorAction SilentlyContinue
+        }   
+
+        Import-Module (Join-Path ((Resolve-Path $PSScriptRoot\..\..).Path) "Modules\xSharePoint")
+        
+      <#  $coreProperty = @{ 
+                           Name = "PersonalInformation"
+                           DisplayName = "Personal Information"
+                        } | Add-Member ScriptMethod Commit {
+                            $Global:xSPUPSPropertyCommitCalled = $true
+                        } -PassThru | Add-Member ScriptMethod Delete {
+                            $Global:xSPUPSPropertyDeleteCalled = $true
+                        } -PassThru
+
+     
+                        
+        
+        #$typeProperties.Add($typeProperty)
+       $subTypePropertyUpdate = @{
+                            Name = "PersonalInformation"
+                            DisplayName = "Personal InformationUpdate"
+                            DisplayOrder =5000
+                        }| Add-Member ScriptMethod Commit {
+                            $Global:xSPUPPropertyCommitCalled = $true
+                        } -PassThru 
+                        #>
```

Comment:
Sure!
Might be able to do it soon. 
Camilo Borges
Sent via iPhone while out and about


> On 11 Feb 2016, at 8:50 PM, Brian Farnhill <notifications@github.com> wrote:
> 
> In Tests/xSharePoint/xSharePoint.xSPUserProfileSection.Tests.ps1:
> 
> > +                        } -PassThru | Add-Member ScriptMethod Delete {
> > +                            $Global:xSPUPSPropertyDeleteCalled = $true
> > +                        } -PassThru
> > +
> > +     
> > +                        
> > +        
> > +        #$typeProperties.Add($typeProperty)
> > +       $subTypePropertyUpdate = @{
> > +                            Name = "PersonalInformation"
> > +                            DisplayName = "Personal InformationUpdate"
> > +                            DisplayOrder =5000
> > +                        }| Add-Member ScriptMethod Commit {
> > +                            $Global:xSPUPPropertyCommitCalled = $true
> > +                        } -PassThru 
> > +                        #>
> Can you pull out this commented stuff if its not needed?
> 
> —
> Reply to this email directly or view it on GitHub.
>